nixos/modules/services/admin/nomachine.nix

```diff
@@ -0,0 +1,233 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+with lib; let
```

Comment:
Avoid using top level `with`s, those can lead to bugs that are very hard to debug

Reply:
I did not not that, removed it.